KifuSF/VerifyDropoffViewController.swift

```diff
@@ -89,11 +89,23 @@ class VerifyDropoffViewController: UIScrollableViewController {
     }
 
     @objc private func reportButtonTapped() {
-        let flaggingViewController = KFCFlagging(flaggableItems: [.flaggedVerificationImage], donation: self.donation)
+        let flaggingViewController = UINavigationController(
+            rootViewController: FlaggingViewController(
+                flaggableItems: flaggableItems,
+                donationToReport: donation
+            )
+        )
         navigationController?.pushViewController(flaggingViewController, animated: true)
```

Comment:
I thought you mentioned to present the FlaggingVc modally?

Reply:
The rootViewController of that nav controller is already presented modally(which is the VerifyDropoffViewController), so I thought it's better to push this screen rather than presenting it.

Reply:
Sounds reasonable